jwst/pixel_replace/pixel_replace_step.py

```diff
@@ -47,12 +48,9 @@ def process(self, input):
             it will be a model containing data arrays with estimated fluxes
             for any bad pixels, now flagged as TO-BE-DETERMINED (DQ bit 7?).
         """
-
         with datamodels.open(input) as input_model:
-
             # Make copy of input to prevent overwriting
             result = input_model.copy()
-
```

Comment:
Somebody (or somebody's automatic IDE) doesn't like blank lines! ;-)  I don't object to them if it helps set off sections of the code to make it more human readable.